editoast/openapi.yaml

```diff
@@ -4240,6 +4445,30 @@ components:
           type: string
           enum:
           - editoast:infra:railjson:WrongRailjsonVersionProvided
+    EditoastNoSuchUserErrorNoSuchUser:
```

Comment:
https://github.com/OpenRailAssociation/osrd/pull/8984#discussion_r1775625670

Reply:
Not sure what I can do about this though (`NoSuchUser::NoUserIdeed`??)

Reply:
Do we need a `enum` since it's only one variant. Won't having a `struct` instead solve the problem? If not, feel free to resolve anyway, I don't see an easy solution.

Reply:
`derive(EditoastError)` only works on `enums` 😢
It's new version will definitely have to though.